src/timeline/TimelineItem.cc

```diff
@@ -61,6 +63,14 @@ TimelineItem::init()
 
         mainLayout_->setContentsMargins(conf::timeline::headerLeftMargin, 0, 0, 0);
         mainLayout_->setSpacing(0);
+
+        // Setting fixed width for checkmark because systems may have a differing width for a
+        // space and the Unicode checkmark.
+        checkmark_ = new QLabel(" ", this);
+        checkmark_->setFixedWidth(fm.width(CHECKMARK));
```

Comment:
The font used here to calculate the width doesn't not have the same size as the one be used on the label. I would suggest to create a temporary font with the correct font size and use that to derive the width.

Reply:
I'm assuming you mean something like this?

```cpp
QFont checkmarkFont;
checkmarkFont.setPixelSize(conf::timeline::font::timestamp);

checkmark_->setFont(checkmarkFont);
checkmark_->setFixedWidth(QFontMetrics{checkmarkFont}.width(CHECKMARK));
...
```

Reply:
Yes, but I'm not sure if the stylesheet with the font size is also needed.

Reply:
I tried it without a stylesheet and it seems to make no difference.